### PR TITLE
New package: Ubiquiti.UniFiNetworkServer version 8.2.93

### LINKED ISSUE #158707: [Package Request]: UniFi Network Server
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://ui.com/download/releases/network-server
Publisher: Ubiquiti
Package Name: UniFi Network Server
Description: The UniFi Network Application, developed by Ubiquiti Inc., is a centralized network management software designed to configure, manage, and monitor UniFi devices and networks. It provides a comprehensive suite of tools for network administrators to handle their network infrastructure efficiently
Package Version: 8.2.93
Installer URL: https://dl.ui.com/unifi/8.2.93/UniFi-installer.exe

They update the UniFi Network Server quite often so would be nice to have it added to winget

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] This PR resolves #158707 

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/158729)

### CODE REVIEW

**Comment by Dragon1573** [comment]
Validated in `amd64` _Windows Sandbox_ instance.

```text
--> Installing WinGet
--> Disabling safety warning when running installer
Tip: you can type 'Update-EnvironmentVariables' to update your environment variables, such as after installing a new software.

--> Configuring Winget
Enabled admin setting 'LocalManifestFiles'.
Enabled admin setting 'LocalArchiveMalwareScanOverride'.


--> Installing the Manifest 8.2.93

Found Ubiquiti UniFi Network Server [Ubiquiti.UniFiNetworkServer] Version 8.2.93
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://dl.ui.com/unifi/8.2.93/UniFi-installer.exe
  ██████████████████████████████   208 MB /  208 MB
Successfully verified installer hash
Starting package install...
Successfully installed

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName                  DisplayVersion Publisher ProductCode    Scope
-----------                  -------------- --------- -----------    -----
Ubiquiti UniFi (remove only)                          Ubiquiti UniFi Machine
```

**Comment by Dragon1573** [comment]
Seems this package requires several TCP/UDP ports opened before first launch. I can't open **Windows Firewall** control panel in _Windows Sandbox_ instance. This may block the initialization of the package.

image

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 886.)

File: manifests/u/Ubiquiti/UniFiNetworkServer/8.2.93/Ubiquiti.UniFiNetworkServer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-06-03
+AppsAndFeaturesEntries:
+  - DisplayName: Ubiquiti UniFi
+Installers:
```
Comment (stephengillie) [inline manifests/u/Ubiquiti/UniFiNetworkServer/8.2.93/Ubiquiti.UniFiNetworkServer.installer.yaml:16]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Verify: 
image
The button didn't become enabled while I was reviewing it, possibly due to:

- This appears to be the [Windows build for MongoDB](https://www.mongodb.com/docs/manual/reference/program/mongod.exe/), and needs `VCRedist`:

image

Application doesn't add version data to Registry:
image

File: manifests/u/Ubiquiti/UniFiNetworkServer/8.2.93/Ubiquiti.UniFiNetworkServer.installer.yaml (hunk 1 above)
Comment (Dragon1573) [reply manifests/u/Ubiquiti/UniFiNetworkServer/8.2.93/Ubiquiti.UniFiNetworkServer.installer.yaml:16]: Seems it's the actual reason. Verified in my `amd64` _Windows Sandbox_.

image
image

**Comment by Dragon1573** [comment]
> Template: msftbot/validationError/urls/domain

Can't get it. 😕

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
- https://github.com/microsoft/winget-pkgs/issues/142854

**Pull request merged**